atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1543,13 +1702,7 @@ public void addGarbageCollectionSentinelValues(TableReference tableRef, Set<Cell
     @Override
     public Multimap<Cell, Long> getAllTimestamps(TableReference tableRef, Set<Cell> cells, long ts) {
         AllTimestampsCollector collector = new AllTimestampsCollector();
-        try {
-            loadWithTs(tableRef, cells, ts, true, collector, deleteConsistency);
-        } catch (UnavailableException e) {
-            throw new InsufficientConsistencyException("Get all timestamps requires all Cassandra nodes to be up and available.", e);
-        } catch (Exception e) {
-            throw Throwables.throwUncheckedException(e);
-        }
+        loadWithTs(tableRef, cells, ts, true, collector, deleteConsistency);
```

Comment:
The full type of `UnavailableException` is `org.apache.cassandra.thrift.UnavailableException`. That's a `TException` which is `Exception`.